tools/msys2_fetch_and_build_all_tx16s.sh

```diff
@@ -15,81 +41,109 @@ do
 	fi
 done
 
+STEP=1
 echo "=== Step $((STEP++)): Creating a directory for EdgeTX ==="
-mkdir ~/edgetx && cd ~/edgetx
+mkdir -p ~/edgetx && cd ~/edgetx
```

Comment:
home should exist, thus -p not required here.

Reply:
how it was created?
I did not had one